config/locales/en.yml

```diff
@@ -1585,8 +1585,8 @@ en:
       anonymous: An anonymous user
       greeting: "Hi,"
       commented:
-        subject_own: "[OpenStreetMap] %{commenter} has commented on one of your notes"
-        subject_other: "[OpenStreetMap] %{commenter} has commented on a note you are interested in"
+        subject_own: "[OpenStreetMap] %{commenter} has commented on your note %{note_id}"
+        subject_other: "[OpenStreetMap] %{commenter} has commented on note %{note_id} you are interested in"
         your_note: "%{commenter} has left a comment on one of your map notes near %{place}."
```

Comment:
You will also need to change the other `subject_own` and `subject_other` translations further down this file - there are separate subjects for "commented" "closed" and "reopened" events.